tests/spec/fs.stat.spec.js

```diff
@@ -92,3 +92,32 @@ describe('fs.stat', function() {
     });
   });
 });
+
+describe('fs.promises.stat', function() {
```

Comment:
Move the following test into the `describe('fs.stat'...)` block.  This makes it easier to add more related tests.